grpc-kotlin-gen/src/main/resources/KtStub.mustache

```diff
@@ -17,7 +17,7 @@ import kotlinx.coroutines.experimental.Deferred
 import kotlinx.coroutines.experimental.channels.LinkedListChannel
 import kotlinx.coroutines.experimental.channels.ReceiveChannel
 import kotlinx.coroutines.experimental.channels.SendChannel
-import kotlinx.coroutines.experimental.launch
+import kotlinx.coroutines.experimental.runBlocking
```

Comment:
Blocking the call to `connect*ToObserver` for the duration of the whole RPC call will lead to threading issues. gRPC expects the `ServerCalls.*Method.invoke()` calls to be asynchronous, specially `StreamingRequestMethod` which is expected to return a `StreamObserver<ReqT>` before the call can proceed.